src/manifests/test_manifest.py

```diff
@@ -62,6 +62,7 @@ class TestManifest(ComponentManifest['TestManifest', 'TestComponents']):
                         "type": "dict",
                         "schema": {
                             "build-dependencies": {"type": "list"},
+                            "topology": {"type": "integer", "required": False},
```

Comment:
I recommend future-proofing this:

```
topology:
    nodes: 2
```

Reply:
+1 . Let's finalize the topology definition even if we're not planning to add support for multi-node clusters. 
Let's also standardize how we'll pass these clusters endpoints as generic params to the `integtest.sh`. 

Maybe define topology something like: 
```
topology: 
    cluster1:
        data_nodes: 1
        master_nodes: 0
    cluster2:
        data_nodes: 1
        master_nodes: 0
```
Since we only support create single node clusters as of now, maybe let's add an assertion for that. 


For cluster endpoints, let's pass it as a json maybe. Somthing like: 
```
{
    "cluster1": {
        "data_nodes": [
             {
                 "endpoint": "localhost",
                 "http_port": 9200,
                 "transport_port": 9300
              },
             {
                 "endpoint": "localhost",
                 "http_port": 9201,
                 "transport_port": 9301
              }
         ]
        "master_nodes": [
             {
                 "endpoint": "localhost",
                 "http_port": 9202,
                 "transport_port": 9302
              }
        ]
    },
    "cluster2": {
        "data_nodes": [
             {
                 "endpoint": "localhost",
                 "http_port": 9203,
                 "transport_port": 9303
              },
             {
                 "endpoint": "localhost",
                 "http_port": 9204,
                 "transport_port": 9304
              }
         ]
        "master_nodes": [
             {
                 "endpoint": "localhost",
                 "http_port": 9205,
                 "transport_port": 9305
              }
        ]
    }
}

```

Reply:
Sure, I will work on finalizing the Topology and the way endpoints are passed.

Reply:
Added this support